README.md

```diff
@@ -14,42 +14,56 @@ Responsible for configuring Scylla during first boot of the instance.
 aws/ami/build_ami.sh
 ```
 
-## Configure ScyllaDB
-When an instance is created using Scylla Machine Image we can configure it during first boot.
-We do this via user-data (AWS)
-### User data format
-The format is JSON and allows us:
-- set any `scylla.yaml` paramater
-- run a script after the configuration is done
-- control whether to start Scylla after the first boot
-
-Defaults are:
+## Scylla AMI user-data Format v2
+
+Scylla AMI user-data should be passed as a json object, as described below
+
+see AWS docs for how to pass user-data into ec2 instances:
+[https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-add-user-data.html](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-add-user-data.html)
+---
+### EC2 User-Data
```

Comment:
Its not clear which of the objects and element are optional (all?) and which are required